docs/ballerina-by-example/examples/byte-i-o/byte-i-o.bal

```diff
@@ -1,72 +1,79 @@
 import ballerina.io;
 
-@Description{value:"This function returns a ByteChannel from a given file location according to the specified file permission (whether the file should be opened for reading/writing)."}
+@Description {value:"This function returns a ByteChannel from a given file location according to the specified file permission (whether the file should be opened for reading/writing)."}
 function getFileChannel (string filePath, string permission) (io:ByteChannel) {
     // Here is how the ByteChannel is retrieved from the file.
     io:ByteChannel channel = io:openFile(filePath, permission);
     return channel;
 }
 
-@Description{value:"This function reads the specified number of bytes from the given channel."}
-function readBytes (io:ByteChannel channel, int numberOfBytes) (blob, int) {
+@Description {value:"This function reads the specified number of bytes from the given channel."}
+function readBytes (io:ByteChannel channel, int numberOfBytes, int offset) (blob, int) {
     blob bytes;
     int numberOfBytesRead;
+    io:IOError readError;
     // Here is how the bytes are read from the channel.
-    bytes, numberOfBytesRead = channel.readBytes(numberOfBytes);
+    bytes, numberOfBytesRead, readError = channel.read(numberOfBytes, offset);
+    if (readError != null) {
+        throw readError.cause;
+    }
     return bytes, numberOfBytesRead;
 }
 
-@Description{value:"This function writes a byte content with the given offset to a channel."}
-function writeBytes (io:ByteChannel channel, blob content, int startOffset) (int) {
+@Description {value:"This function writes a byte content with the given offset to a channel."}
+function writeBytes (io:ByteChannel channel, blob content, int startOffset, int size) (int) {
+    int numberOfBytesWritten;
+    io:IOError writeError;
     // Here is how the bytes are written to the channel.
-    int numberOfBytesWritten = channel.writeBytes(content, startOffset);
+    numberOfBytesWritten, writeError = channel.write(content, startOffset, size);
```

Comment:
Shall we use var here?

Reply:
As discussed with @anupama-pathirage we will be leaving the samples as it is to preserve readability instead of using var